Core/ComponentBaseClasses/elxTransformBase.h

```diff
@@ -192,6 +192,12 @@ class TransformBase :
   typedef typename ITKBaseType::InputPointType  InputPointType;
   typedef typename ITKBaseType::OutputPointType OutputPointType;
 
+  /** Typedef's for TransformPointsAllPoints. */
+  typedef itk::Vector<
+    float, FixedImageDimension >                      VectorPixelType;
```

Comment:
Is the coord type guaranteed to be float always? @stefanklein

Reply:
Good remark, I could have chosen double as well. I didn't check what is most in line with elastix. Preferably I would make it a template argument, but that far from trivial in the elastix framework. Other constructions would get complicated as well, so I think we just have to choose between float or double. What is your preference?

Reply:
I think float is good enough since it has more than enough precision (1mm x 10^-7 ~ 1nm :D) and takes up half as much ram :)